### ISSUE TITLE
Custom location of `index.html`

### ISSUE THREAD

**Issue by schlunma** [ESMValGroup/ESMValCore#2274]
**Is your feature request related to a problem? Please describe.**

To setup a continuous monitoring of climate model simulations, it would be helpful if one could specify a custom location for the `index.html` file (with working links to the output of course).

**Would you be able to help out?**

Yes, will work on that now.

**Comment by valeriupredoi** [comment]
`index.html` for what, bud? :grin:

**Comment by schlunma** [comment]
I am referring to the `index.html` file that is written by ESMValTool :grin: 

While testing this, I found that it is not really useful to have a custom location for `index.html`. The main benefits on this would be on a remote server that can always use the same root path. However, when using a custom location for `index.html`, we need to reference the figures with absolute paths, which is not possible on a web server due to obvious security reasons.

I found  that is more helpful to setup a simple web server with port forwarding in this case. Since this might be relevant for other users as well, I will open a PR on this that describes this procedure :+1:
